src/workerd/api/actor.h

```diff
@@ -143,6 +143,13 @@ class DurableObjectNamespace: public jsg::Object {
       CompatibilityFlags::Reader featureFlags);
   // Gets a durable object by ID or creates it if it doesn't already exist.
 
+  jsg::Ref<DurableObject> getExisting(
+      jsg::Ref<DurableObjectId> id,
+      jsg::Optional<GetDurableObjectOptions> options,
+      CompatibilityFlags::Reader featureFlags);
+  // Experimental. Gets a durable object by ID if it already exists. Currently, gated for use
+  // by cloudflare only.
```

Comment:
Very much a nit: `:s/cloudflare/Cloudflare`

Reply:
I wouldn't mention cloudflare explicitly here at all. I'd just say something like "experimental feature not currently implemented in workerd" or something.

Reply:
@xortive I don't think this was addressed?